Source/PluginProcessor.cpp

```diff
@@ -292,7 +375,13 @@ void SoomplerAudioProcessor::setTransportSource(AudioFormatReader* reader)
 {
     auto newSource = std::make_unique<AudioFormatReaderSource>(reader, true);
     transportSource.setSource(newSource.get(), 0, nullptr, reader->sampleRate);
-    readerSource = std::move(newSource);
+    readerSource.reset(newSource.release());

```

Comment:
use move semantics (read about std::move) and documentation for unique_ptr

Reply:
didn't catch you, where i supposed to use `std::move`?